src/main/webapp/config.js

```diff
@@ -1,3 +1,43 @@
+let initialFormData = {};
+
+let webHookUrlInput;
+let siteInput ;
+
+window.addEventListener('DOMContentLoaded', () => {
```

Comment:
Jenkins doesnt play great with the way we have the form :sigh: 
Since all its validations are done on the backend i needed to manually invoke the "changed" event on the inputs, bit of a hassle but works a treat